doc/how-to/microcloud-with-terraform.md

```diff
@@ -0,0 +1,62 @@
+# Single Node Microcloud deployment (Nested VMs)
+
+## Prerequisites
+
+- Single Ubuntu node with LXD installed
+- A network of type bridged on the host, e.g. from lxc network list
+```
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+|      NAME      |   TYPE   | MANAGED |      IPV4       |           IPV6            | DESCRIPTION | USED BY |  STATE  |
++----------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
+| br0            | bridge   | NO      |                 |                           |             | 17      |         |
```

Comment:
Can we use LXD here to create another managed bridge that will be used later for OVN? Something like [this](https://github.com/roosterfish/microcloud-roadshow/blob/main/deploy.sh#L44).

It would have the added benefit that you can just launch Terraform without caring much about what is already setup.